h5py/api_functions.txt

```diff
@@ -290,6 +290,7 @@ hdf5:
   herr_t    H5Pget_sym_k(hid_t plist, unsigned int *ik, unsigned int *lk)
   herr_t    H5Pset_istore_k(hid_t plist, unsigned int ik)
   herr_t    H5Pget_istore_k(hid_t plist, unsigned int *ik)
+  herr_t    H5Pset_file_space_strategy(hid_t fcpl, H5F_fspace_strategy_t strategy, hbool_t persist, hsize_t threshold)
```

Comment:
[The docs say](https://confluence.hdfgroup.org/display/HDF5/H5P_SET_FILE_SPACE_STRATEGY) this function was added in 1.10.1. As we still support building against h5py 1.8, you'll need to add the version number in this listing (see examples elsewhere in the same file).